ggml.h

```diff
@@ -451,7 +451,11 @@ extern "C" {
 
         void * extra; // extra things e.g. for ggml-cuda.cu
 
-        char padding[4];
+        // temp - used by allocator
+        int n_children;
+        int n_views;
```

Comment:
Currently adds these fields to `ggml_tensor`, but they could be moved to a hash table in the allocator instead.